src/status_im/chat/handlers.cljs

```diff
@@ -100,7 +100,7 @@
 (register-handler :cancel-command
   (fn [{:keys [current-chat-id] :as db} _]
     (-> db
-        (dissoc :canceled-command)
+        (assoc :canceled-command nil)
```

Comment:
What is the rational for those changes? Is this comething that should be defined as part of best practises?